### DISCUSSION TITLE
[Support] dev 文件变化后总是无法热加载

### DISCUSSION THREAD

**Discussion by hooliy** [midwayjs/midway#3731] [Q&A]
### Describe the problem（描述问题）

![image](https://github.com/midwayjs/midway/assets/3368691/3cb4c3a1-86c9-406d-acf9-43089b16597a)

`
"dev": "cross-env NODE_ENV=local mwtsc --watch --run @midwayjs/mock/app.js",
`

yarn dev:  文件变化后总是无法热加载，有时候加载成功也会报端口占用的错误。

是怎么回事？

### Midway Versions（Midway 版本）

@latest

**Comment by czy88840616** [comment]
文件变了，会有变了的 log 的吧。

[0:33:57] 1 Files has been changed.

这样的。

**Reply by hooliy** [reply]
抽风的次数比较多吧

**Reply by czy88840616** [reply]
文件修改的监听是chokidar做的，估计还是 windows 的兼容性有问题。

**Reply by hooliy** [reply]
好吧
